server/views/pages/goal/unarchive/index.njk

```diff
@@ -37,7 +37,7 @@
 
           {{ govukButton({
             type: "a",
-            href: "/plan/" + prisonerSummary.prisonNumber + "/view/archived-goals",
+            href: "/plan/" + prisonerSummary.prisonNumber + "/view/goals#archived-goals",
```

Comment:
Fix the url to redirect to the new archived goals tab instead of old archived goals page.

Reply:
Great spot 👍